src/Components/ModalWithChildren.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Modal } from 'react-bootstrap';
+
+interface loginProps {
```

Comment:
Copy pasta issue!

Should be named ModalWithChildrenProps

Reply:
Modal with Children has been cancelled.